visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -129,15 +135,28 @@ void MotionPlanningFrame::computePlanButtonClicked()
 void MotionPlanningFrame::computeExecuteButtonClicked()
 {
   if (move_group_ && current_plan_)
-    move_group_->execute(*current_plan_);
+  {
```

Comment:
Please keep the background jobs in there.
This is common across RViz and whenever RViz hangs when operating a robot over wifi, I can be sure that someone did *not* start a background job.
In that case the problem is not the computation time of the RPC, but the network latency involved.